### ISSUE TITLE
Error Code 253

### ISSUE THREAD

**Issue by Lettucecake** [MatthewPierson/Vieux#2]
iPhone 5s, in DFU mode, iOs 12.4.3, downgrade to 10.3.3 failed. Tried more than once, have dependencies
MacOS Catalina

Log:

```
Vieux - A tool for 32/64 Bit OTA downgrades
Still in BETA so expect issues/broken things
If you are using a 64 Bit device then connect it in DFU Mode
If you are using a 32 Bit device then just have it connected in normal mode
Starting IPSW unzipping
Continuing...
IPSW found at given path...
Cleaning up old files...
Files cleaned.
Unzipping..
Found: CPID:8960 CPRV:11 CPFM:03 SCEP:01 BDID:02 ECID:0000060E8B4347C8 IBFL:1C SRTG:[iBoot-1704.10] PWND:[checkm8]
Device is already in pwned DFU Mode. Not executing exploit.
Exploit worked!
*** SecureROM Signature check remover by Linus Henze ***
Applying patches...
Patches have already been applied. Exiting.

Starting iBSS/iBEC patching
Looks like you are downgrading an iPhone 5s to 10.3.3!
Patched iBSS/iBEC
About to re-build IPSW
Entering PWNREC mode...
Getting SHSH...
Restoring...
Note that errors about 'BbSkeyId', 'FDR Client' and 'BasebandFirmware Node' are not important, just ignore them and only report errors that actually stop the restore.
ERROR..
Return code: 253
Restore Failed.
Please try again and report the error + full logs if it persists.
Exiting...
```

**Comment by MatthewPierson** [comment]
Have you rebooted the device and tried again? It may just be an issue with Catalina, Catalina breaks a lot of things sadly.

**Comment by Lettucecake** [comment]
I have tried rebooting. Just for the sake of testing, I will try ra1nstorm. Thank you for your help, one of the most responsive devs ever!

**Comment by suenky91** [comment]
Same problem on Catalina, I even tried to disable SiP

**Comment by Lettucecake** [comment]
Resolved???? - I got my iPhone 5 to restore to 8.4.1 fine on Mojave using KVM.
